upgrading/manual_upgrades.adoc

```diff
@@ -509,7 +509,8 @@ updated.
 
 During a cluster upgrade, and on every restart of any master, the 
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. 
+cluster roles] are automatically reconciled, or updated to contain all the
```

Comment:
This seems to imply that it does one or the other - perhaps we can say
"are automatically reconciled to restore any missing permissions.